src/Pagination/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { Pagination as AntPagination, PaginationProps as AntPaginationProps } from 'antd'
+import { CaretLeft, CaretRight } from 'phosphor-react'
+import './style.css'
+
+export type PaginationProps = AntPaginationProps & {
+  LeftIcon: React.ReactNode
+  RightIcon: React.ReactNode
+}
+
+const createItemRender = (LeftIcon: React.ReactNode, RightIcon: React.ReactNode) => (
```

Comment:
can call it `itemRender`. `createItemRender` means this function will return another function